services/QuillLMS/app/services/stripe_integration/webhooks/checkout_session_completed_event_handler.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module StripeIntegration
+  module Webhooks
+    class CheckoutSessionCompletedEventHandler < EventHandler
+      def run
+        stripe_checkout_session&.update!(expiration: DateTime.now.utc)
```

Comment:
Should we consider just deleting this record instead of keeping it around with an expiration assigned?  Do we actually want this for record-keeping purposes?  Or am I wrong in thinking that deleting the record would give us the same result in terms of duplicate-protection?

Reply:
@anathomical Good question.  I wrestled with this decision myself.  My first pass at this, I just deleted the record like you suggested `stripe_checkout_session`.  However, when troubleshooting a specific edge case (and potentially future ones), I found it more useful to keep the object around.  

That being said, I do agree though that these probably add little value long term.  Maybe a cron job  to clean these out maybe weekly/monthly?

Reply:
@anathomical I've moved it back to just `destroy!`.  Thank you for the nudge